src/mcse.jl

```diff
@@ -1,9 +1,13 @@
-#################### Monte Carlo Standard Errors ####################
-
 """
-    mcse(x::AbstractVector{<:Real}; method::Symbol=:imse, kwargs...)
+    mcse(x::AbstractVector{<:Real}, method::Symbol=:imse; kwargs...)
+
+Return Monte Carlo Standard Errors of samples `x`.
+Here, `method` describes how to estimate the errors; possible options are:
+
+- `:bm` for batch means
+- `:imse` for the integrated mean-squared (prediction) error
+- `:ipse` for the initial positive sequence estimator
```

Comment:
At some point we should explain it more clearly but this is definitely an improvement :slightly_smiling_face: